configure.ac

```diff
@@ -405,8 +405,8 @@ if test "$enable_media_studio_va" = "yes"; then
         [Defined to 1 to enable H265 encoding when --enable-media-studio-va="yes"])
 fi
 
-AC_SUBST([AM_CFLAGS], ["-g -O2 -Wall -Wno-unused-function -Wno-cpp -Werror"])
-AC_SUBST([AM_CXXFLAGS], ["-g -O2 -Wall -Wno-unused-function -Wno-cpp -Wno-missing-braces -Werror"])
+AC_SUBST([AM_CFLAGS], ["-g -O2 -Wall -Wno-unused-function -Wno-cpp -Werror -D_FORTIFY_SOURCE=2 -fPIE -fPIC -fstack-protector"])
+AC_SUBST([AM_CXXFLAGS], ["-g -O2 -Wall -Wno-unused-function -Wno-cpp -Wno-missing-braces -Werror -z noexecstack -z relro -z now -pie"])
```

Comment:
I think you misunderstood me.  Please add AM_LDFLAGS and AM_CPPFLAGS with your new compiler options.  Don't append them to AM_CFLAGS and AM_CXXFLAGS... those are the wrong variables.

Reply:
Sorry for my mistake, I uploaded it wrongly and I have modified it just now.